### PR TITLE
Typo fix

### PR DESCRIPTION
**Pull request by DelazJ**

### CODE REVIEW

File: src/ui/qgsvectorlayerpropertiesbase.ui [hunk 1]
```diff
@@ -2225,7 +2225,7 @@ border-radius: 2px;</string>
             <item row="0" column="0">
              <widget class="QLabel" name="label_1">
               <property name="text">
-               <string>Auxiliary storage tables can contain additional data that should only belong to the project file. For instance, specific location or rotation for labels. Auxiliary data are saved in qgd files. New fields can be added from any data-defined widget when needed. Be aware that this information will NOT be saved in you datasource but only in the project file.</string>
+               <string>Auxiliary storage tables can contain additional data that should only belong to the project file. For instance, specific location or rotation for labels. Auxiliary data are saved in qgd files. New fields can be added from any data-defined widget when needed. Be aware that this information will NOT be saved in your datasource but only in the project file.</string>
```
Comment (DelazJ) [inline src/ui/qgsvectorlayerpropertiesbase.ui]: I wonder if `Be aware that this information will NOT be saved in **THE** datasource  but only in the project file.` isn't better.
And is it `data source` or `datasource`?

Comment (nyalldawson) [reply src/ui/qgsvectorlayerpropertiesbase.ui]: *the* is better than *your*.  Should be "data source" in this context.

File: src/providers/ogr/qgsgeopackagedataitems.cpp [hunk 2]
```diff
@@ -519,7 +519,7 @@ void QgsGeoPackageCollectionItem::addTable()
   }
   else
   {
-    QgsDebugMsg( QStringLiteral( "Cannot add Table: connection %1 does not exists or the path is empy!" ).arg( connName ) );
```
Comment (nyalldawson) [inline src/providers/ogr/qgsgeopackagedataitems.cpp]: can you add empy to the spell check file ?

**Comment by DelazJ** [comment]
@nyalldawson done!

**Pull request merged**